messages/jsonschema/scripts/templates/perl.pm.erb

```diff
@@ -131,12 +131,12 @@ __END__
 
 =head1 LICENSE
 
-Please see the included LICENSE.txt for the canonical version. In summary:
+Please see the included LICENSE for the canonical version. In summary:
 
 The MIT License (MIT)
 
-Copyright (c) 2021 Erik Huelsmann
-
+  Copyright (c) 2021 Erik Huelsmann
+  Copyright (c) 2021 Cucumber Ltd
```

Comment:
The code on which this implementation is loosely based (the ruby messages implementation), lists an MIT license with Cucumber Ltd copyrights.

Reply:
As https://github.com/cucumber/common/pull/1670#discussion_r683503958 and as discussed during our last community meeting: this is good for now as part of this PR.